src/components/Modal/Modal.vue

```diff
@@ -79,10 +79,12 @@
 					<h2>Please enter your name</h2>
 					<TextField label="First Name" :value.sync="firstName" />
 					<TextField label="Last Name" :value.sync="lastName" />
-					<Button
+					<ButtonVue
 						:disabled="!this.firstName || !this.lastName"
 						@click="closeModal"
-						type="primary">Submit</Button>
+						type="primary">
+						Submit
```

Comment:
does this need translation?

Reply:
No, it's just in the docs, not in the component.